scripts/kconfig/kconfig.py

```diff
@@ -195,20 +197,16 @@ def write_kconfig_filenames(paths, root_path, output_file_path):
     # relative paths, and it should not have duplicates.
 
     # Remove duplicates
-    paths_uniq = set(paths)
+    paths_uniq = set([normpath(abspath(join(root_path, path))) for path in paths])
```

Comment:
This does more than remove duplicates so the comment needs updating. Could possibly be removed.

Reply:
I agree, the name of the variable tells it all.

Reply:
The fact that we normalize paths, and what that means, should be commented somewhere.